src/main/java/com/jonathanpli/exceptionnull/controller/IndexController.java

```diff
@@ -3,26 +3,54 @@
 import org.springframework.stereotype.Controller;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.servlet.ModelAndView;
 
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
 /**
  * Routes root and error pages to their respective URLs. Also contains the fallback case.
  */
 @Controller
 @RequestMapping(value = "/")
 public class IndexController {
+	// TODO(#15): Remove when proper login/logout is implemented.
+	private static final String PASSWORD = "omg it's a password!";
+	public static final String SESSION_IS_LOGGED_IN = "user-is-logged-in";
 
 	/**
 	 * Fallback and index page routing.
 	 * @return
 	 */
-	@RequestMapping(value = "*", method = { RequestMethod.GET, RequestMethod.POST } )
+	@RequestMapping(value = "*")
 	public ModelAndView getIndex() {
 		ModelAndView mav = new ModelAndView();
-
-		mav.addObject("title", "Exception Null");
-		mav.addObject("description", "Exception Null is the new coding platform to teach you how to become a better programmer!");
 		mav.setViewName("index");
 		return mav;
 	}
+
+	// TODO(#15): Remove when proper login/logout is implemented.
+	/**
+	 * A simple login
+	 */
+	@RequestMapping(value = "/login")
+	public ModelAndView login(
+			@RequestParam(value = "login") String login,
+			HttpServletRequest request) {
+		if (login.equals(PASSWORD)) {
+			request.getSession().setAttribute(SESSION_IS_LOGGED_IN, "");
```

Comment:
empty string counts as having the attribute set? Interesting. Could we set it to true for clarity?